_build_cfg.yml

```diff
@@ -118,7 +118,9 @@ Topics:
         File: throttling
       - Name: Source Control Management
         File: scm
-
+  - Name: Revision History
+    File: rev_history
```

Comment:
I'm guessing these are vestigial from an earlier local test run, causing the current build failure.